### PR TITLE
Nextcloud: ansible-ize config

### LINKED ISSUE #11: Nextcloud: Deploy config.php with ansible (at least partly)

### PR DESCRIPTION
**Pull request by marcelklehr**
fixes #11 

Only disadvantage here, is that the current version needs to be kept up-to-date :/ I'm open for suggestions, though :)

### CODE REVIEW

File: host_vars/de-staging/nextcloud.yml [hunk 1]
```diff
@@ -0,0 +1 @@
+nextcloud_directory_data: "{{nextcloud_directory_root}}/data"
```
Comment (zonradkuse) [inline host_vars/de-staging/nextcloud.yml]: this looks recursive - does it actually work?

File: roles/nextcloud/templates/config.php [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  'passwordsalt' => '{{nextcloud_passwordsalt}}',
+  'secret' => '{{nextcloud_secret}}',
+  'trusted_domains' =>
+  [
+	0 => 'cloud.{{domain}}',
+	1 => 'owncloud.{{domain}}',
+  ],
+  'datadirectory' => '/var/www/html/data',
+  'dbtype' => 'mysql',
+  'version' => '16.0.1.1',
```
Comment (zonradkuse) [inline roles/nextcloud/templates/config.php]: can we (re)use some variable?

for example specifying the version once via major and minor version and by this we determine docker image and what to put here

File: host_vars/de-staging/nextcloud.yml (hunk 1 above)
Comment (zonradkuse) [reply host_vars/de-staging/nextcloud.yml]: oh, it doesn't - nvm

File: roles/nextcloud/templates/config.php (hunk 2 above)
Comment (marcelklehr) [reply roles/nextcloud/templates/config.php]: that could work, unfortunately we don't usually know the build number

Comment (zonradkuse) [reply roles/nextcloud/templates/config.php]: Okay, nextcloud has a docker tag for 16.0.1 - are you sure that 16.0.1.1 is correct? 

if it is not and we can use it, I would suggest using a version variable:
```
nextcloud_version : '16.0.1'
```
then
```
'version' => '{{ nextcloud_version }}',
```
and add tag to docker compose file.

Comment (marcelklehr) [reply roles/nextcloud/templates/config.php]: ah, wait, the problem is that 'version' is managed by the update script in nextcloud, so if we put a new version in here, the update script will think it isn't needed.

Comment (marcelklehr) [reply roles/nextcloud/templates/config.php]: we could read out the current value and splice it into the template

Comment (zonradkuse) [reply roles/nextcloud/templates/config.php]: aha! that's inconvenient. can we trigger the update automatically?

**Pull request merged**